doc/Command-Reference.md

```diff
@@ -9285,6 +9318,115 @@ This command is to add or delete a member port into the already created vlan.
   This command will add Ethernet4 as member of the vlan 100.
   ```
 
+**config vlan member add/del -m -e <vlan_ids> <member_portname>**
+
+This command is to add or delete multiple vlan members into port.
+
+_NOTE: This command is only for adding tagged vlan members into port._
+
+- Usage:
+
+  ```
+  config vlan member add/del [-m|--multiple] [-e|--except_flag] <vlan_ids> <member_portname>
+  ```
+
+- Example (Add the VLAN "Vlan100,Vlan200,Vlan300" in one command on port if they don't exist as members):
+
+  ```
+  admin@sonic:~$ sudo config vlan member add -m  100,200,300 Ethernet0

```

Comment:
Suggest to include range support as well with -m option instead of new range command.
For example,
sudo config vlan member add -m 100-300 Ethernet4

Reply:
Updated as suggested